app/controllers/concerns/requirements_version.rb

```diff
@@ -0,0 +1,15 @@
+require "rubygems/dependency"
```

Comment:
Much clener now :crystal_ball: Thanks @nakul9597

Reply:
Thank you for that help :D